File: cpp/MathCode/matrix.cpp
#include <iostream>
using namespace std;

int main(){
    
    int mt[3][3];
    
    for(int i = 0; i < 3 ; ++i){
        for(int j = 0; j < 3 ; ++j){
            cout << "Input matrix[" << i << "][" << j << "] : ";
            cin >> mt[i][j];
        }
    }
    
  cout << "Matrix: \n";
      for(int i = 0; i < 3 ; ++i){
        for(int j = 0; j < 3 ; ++j){
            cout << mt[i][j] << " ";
        }
        cout << "\n";
    }
    
    return 0;
}

File: python/matrix.py

# Declare mt 3x3 matrix
mt = [[0 for _ in range(3)] for _ in range(3)]

# Input values into the matrix
print("Enter 9 integers for the matrix:")
for i in range(3):
    for j in range(3):
        mt[i][j] = int(input(f"Enter value for mt[{i}][{j}]: "))

# Output the matrix
print("The 3x3 matrix is:")
for i in range(3):
    for j in range(3):
        print(mt[i][j], end=' ')
    print()  # Newline after each row
